frontend/src/App.jsx

```diff
@@ -1,13 +1,13 @@
 import React, { useState, useEffect } from 'react';
 import { Routes, Route, Navigate, useNavigate } from 'react-router-dom';
-import Header from './components/header/Header';
-import Dashboard from './pages/spotnet/dashboard/Dashboard';
-import Footer from './components/Footer/Footer';
-import SpotnetApp from './pages/spotnet/spotnet_app/SpotnetApp';
-import Login from "./pages/Login";
-import Form from "./pages/forms/Form";
-import { connectWallet, logout } from './utils/wallet';
-import { saveTelegramUser, getTelegramUserWalletId} from './utils/telegram';
+import Header from 'components/Header/Header';
```

Comment:
This should be `components/header/Header` with `header` starting from a lowercase letter, currently this line seems to cause build error and failed CI